data/tlg0062/tlg001/tlg0062.tlg001.perseus-grc2.xml

```diff
@@ -92,20 +94,20 @@
+<p> ἤδη δὲ καὶ περὶ τοῦ ἀφεῖναι τὴν ἀρχὴν καὶ καταθέσθαι τὴν δυναστείαν ἐσκοπούμην, ὅπως μόνον ἀσφαλῶς παύσαιτο ἄν τις ἐννοῶν, ἐπεὶ τό γε ἄρχειν αὐτὸ καὶ πάντα πράττειν ἐπαχθὲς ἤδη καὶ σὺν φθόνῳ καματηρὸν ἐδόκει μοι εἶναι· τὸ δʼ ὅπως μηκέτι τοιαύτης τινὸς θεραπείας δεήσεται ἡ πόλις, τοῦτʼ ἐζήτουν ἔτι. κἀγὼ μὲν ὁ ἀρχαῖος περὶ ταῦτα εἶχον, οἱ δὲ ἤδη τε συνίσταντο ἐπʼ ἐμὲ καὶ περὶ τοῦ τρόπου τῆς ἐπιβουλῆς καὶ ἀποστάσεως ἐσκοποῦντο καὶ συνωμοσίας συνεκρότουν καὶ ὅπλα ἤθροιζον καὶ χρήματα ἐπορίζοντο καὶ τοὺς ἀστυγείτονας ἐπεκαλοῦντο καὶ εἰς τὴν <pb n="v.1.p.8"/> Ἑλλάδα παρὰ Λακεδαιμονίους καὶ Ἀθηναίους ἐπρεσβεύοντο· <corr>ἃ</corr> μὲν γὰρ περὶ ἐμοῦ αὐτοῦ, εἰ ληφθείην, ἐδέδοκτο ἤδη αὐτοῖς <corr>καὶ</corr> <corr>ὅπως</corr> με αὐτοχειρίᾳ διασπάσεσθαι ἠπείλουν καὶ ἃς κολάσεις ἐπενόουν, δημοσίᾳ στρεβλούμενοι ἐξεῖπον. τοῦ μὲν δὴ μηδὲν παθεῖν τοιοῦτον οἱ θεοὶ αἴτιοι φωράσαντες τὴν ἐπιβουλήν, καὶ μάλιστά γε ὁ Πύθιος ὀνείρατά τε προδείξας καὶ τοὺς μηνύσοντας ἕκαστα ἐπιπέμπων.</p>
 </div>
    <div type="textpart" subtype="section" xml:base="urn:cts:greekLit:tlg0062.tlg001.perseus-grc2:1" n="5">
-    <p rend="align(indent)"> ἐγὼ δὲ ἐνταῦθα ἤδη ὑμᾶς, ὦ Δελφοί, ἐπὶ τοῦ αὐτοῦ δέους νῦν τῷ λογισμῷ γενομένους ἀξιῶ περὶ τῶν τότε πρακτέων μοι συμβουλεῦσαι, ὅτε ἀφύλακτος ὀλίγου δεῖν ληφθεὶς ἐζήτουν τινὰ σωτηρίαν περὶ τῶν παρόντων. πρὸς ὀλίγον οὖν τῇ γνώμῃ ἐς Ἀκράγαντα παρʼ ἐμὲ ἀποδημήσαντες καὶ ἰδόντες τὰς παρασκευὰς αὐτῶν καὶ τὰς ἀπειλὰς ἀκούσαντες εἴπατε τί δεῖ ποιεῖν; φιλανθρωπίᾳ χρῆσθαι πρὸς αὐτοὺς ἔτι καὶ φείδεσθαι καὶ ἀνέχεσθαι ὅσον αὐτίκα μελλήσοντα πείσεσθαι τὰ ὕστατα; μᾶλλον δὲ γυμνὴν ἤδη ὑπέχειν τὴν σφαγὴν καὶ τὰ φίλτατα ἐν ὀφθαλμοῖς ὁρᾶν ἀπολλύμενα; ἢ τὰ μὲν τοιαῦτα πάνυ ἠλιθίου τινὸς εἶναι, γενναῖα δὲ καὶ ἀνδρώδη διανοηθέντα καὶ χολὴν ἔμφρονος καὶ ἠδικημένου ἀνδρὸς ἀναλαβόντα μετελθεῖν ἐκείνους, ἐμαυτῷ δὲ ἐκ τῶν ἐνόντων τὴν ἐς τὸ ἐπιὸν ἀσφάλειαν παρασχεῖν; ταῦτʼ οἶδʼ ὅτι συνεβουλεύσατε ἄν.</p>
+    <p rend="align(indent)"> ἐγὼ δὲ ἐνταῦθα ἤδη ὑμᾶς, ὦ Δελφοί, ἐπὶ τοῦ αὐτοῦ δέους νῦν τῷ λογισμῷ γενομένους ἀξιῶ περὶ τῶν τότε πρακτέων μοι συμβουλεῦσαι, ὅτε ἀφύλακτος ὀλίγου δεῖν ληφθεὶς ἐζήτουν τινὰ σωτηρίαν περὶ τῶν παρόντων. πρὸς ὀλίγον οὖν τῇ γνώμῃ ἐς Ἀκράγαντα παρʼ ἐμὲ ἀποδημήσαντες καὶ ἰδόντες τὰς παρασκευὰς αὐτῶν καὶ τὰς ἀπειλὰς ἀκούσαντες εἴπατε τί δεῖ<note type="footnote" n="v.1.p.8.n.1">δεῖ MSS.: ἔδει Cobet.</note> ποιεῖν; φιλανθρωπίᾳ χρῆσθαι πρὸς αὐτοὺς ἔτι καὶ φείδεσθαι καὶ ἀνέχεσθαι ὅσον αὐτίκα μελλήσοντα πείσεσθαι τὰ ὕστατα; μᾶλλον δὲ γυμνὴν ἤδη ὑπέχειν τὴν σφαγὴν καὶ τὰ φίλτατα ἐν ὀφθαλμοῖς ὁρᾶν ἀπολλύμενα; ἢ τὰ μὲν τοιαῦτα πάνυ ἠλιθίου τινὸς εἶναι, γενναῖα δὲ καὶ ἀνδρώδη διανοηθέντα καὶ χολὴν ἔμφρονος καὶ ἠδικημένου ἀνδρὸς ἀναλαβόντα μετελθεῖν ἐκείνους, ἐμαυτῷ δὲ ἐκ τῶν ἐνόντων τὴν ἐς τὸ ἐπιὸν ἀσφάλειαν παρασχεῖν; ταῦτʼ οἶδʼ ὅτι συνεβουλεύσατε ἄν.</p>
```

Comment:
We don't generally number notes if they are placed after the word.

you need a note language attribute and to fix the internal language in the notes.

So if the notes are in English or Latin, then the Greek in the note needs tagging. (or vice versa)

`<note type="footnote" n="v.1.p.8.n.1">δεῖ MSS.: ἔδει Cobet.</note>`

vs

`<note type="footnote" n="v.1.p.8.n.1" xml:lang="eng"><foreign xml:lang="grc">δεῖ</foreign> MSS.: <foreign xml:lang="grc">ἔδει</foreign> Cobet.</note>`